### PR TITLE
release 0.11.6

### PR DESCRIPTION
**Pull request by khoroshevskyi**
- Updated readme
- Added namespace stat endpoint
- Fixed 292
- Updated

### CODE REVIEW

File: pephub/routers/api/v1/namespace.py [hunk 1]
```diff
@@ -440,3 +445,15 @@ async def get_namespace_information(
...
+@namespaces.get(
+    "/stats",
+    summary="Get statistics about each namespace",
+    response_model=NamespaceStats,
+)
+async def get_namespace_information(
+    agent: PEPDatabaseAgent = Depends(get_db),
+    namespace: Optional[str] = None,
+):
+    return agent.namespace.stats(namespace=namespace)
```
Comment (nleroy917) [inline pephub/routers/api/v1/namespace.py]: What if namespace is one that doesnt exist? Do we need to catch for `404`s?

File: web/src/components/tables/sample-table.tsx [hunk 2]
```diff
@@ -44,6 +45,12 @@ export const SampleTable: FC<Props> = ({ data, readOnly = false, onChange, heigh
           height={height || tableHeight}
           readOnly={readOnly}
           colHeaders={true}
+          colWidths={200}
+          rowHeights={23}
+          renderer={(instance, td, row, col, prop, value, cellProperties) => {
```
Comment (nleroy917) [inline web/src/components/tables/sample-table.tsx:50]: Cool. This looks minimal changes, so if it works for you locally I trust to deploy it

**Review by nleroy917 — APPROVED** [review]
Nice. Just that one thing on the potential for a `404`

**Review by nleroy917 — APPROVED** [review]
Just that one thing on the potential for a `404` if someone submits a namespace that doesn't exist. I trust that the handsontable stuff is correct if it works locally!

File: web/src/components/tables/sample-table.tsx (hunk 2 above)
Comment (khoroshevskyi) [reply web/src/components/tables/sample-table.tsx:50]: It works, but also can be improved in the future

File: pephub/routers/api/v1/namespace.py (hunk 1 above)
Comment (khoroshevskyi) [reply pephub/routers/api/v1/namespace.py]: done

**Pull request merged**